.github/workflows/pr.yml

```diff
@@ -0,0 +1,31 @@
+name: Test Pull Request
+
+on:
+    pull_request:
+        types:
+            - opened
+            - reopened
+            - synchronize
+    push:
+    workflow_dispatch:
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        name: Run js tests
+        steps:
+            -   name: Checkout code
+                uses: actions/checkout@v3
+            -   name: Setup node
+                uses: actions/setup-node@v3
+                with:
+                    node-version-file: ".nvmrc"
+                    cache: "npm"
+            -   name: Install dependencies
+                run: npm ci
+            -   name: Run tests
+                run: |
+                    npm run lint
+                    npm run build
+            -   name: Upload coverage to Codecov
+                uses: codecov/codecov-action@v3
```

Comment:
Codecov is not configured for this repo, also we do not have any test to check the code coverage. Can be safely removed :)